### ISSUE TITLE
Font resizing bug

### ISSUE THREAD

**Issue by 0x-r4bbit** [maccman/wysiwyg#1]
After selecting a text and clicking 'Large', the text becomes large. If you now click 'Medium' the text doesn't become medium, it becomes even larger. And that happens all the time when switching between large and medium.

Not sure if this is the right behavior.

**Comment by lukasoppermann** [comment]
I agree, instead of adding h2 and h3 all the time it should actually replace it or if you click it again undo the h2.
